Source/WebCore/loader/CrossOriginPreflightChecker.cpp

```diff
@@ -123,7 +123,7 @@ void CrossOriginPreflightChecker::startPreflight()
     options.initiatorContext = m_loader.options().initiatorContext;
 
     CachedResourceRequest preflightRequest(createAccessControlPreflightRequest(m_request, m_loader.securityOrigin(), m_loader.referrer()), options);
-    preflightRequest.setInitiator(m_loader.options().initiator);
+    preflightRequest.setInitiator(AtomString { m_loader.options().initiator });
```

Comment:
Seems like a strange boundary here. Why are `CachedResourceRequest` using `AtomString`, but not loader options?

Reply:
The issue is that ThreadableLoaderOptions is passed across threads and has an isolatedCopy() member functions. As a result, it cannot have a data member that is an AtomString.